src/to-have-style.js

```diff
@@ -11,10 +12,6 @@ function isSubset(expected, received) {
   );
 }
 
-function mergeAllStyles(styles) {
-  return mergeAll(styles.flat());
```

Comment:
Seems like `mergeAll` is no longer used anywhere else, pls remove it with related tests.

Reply:
Done! without `mergeAll` it was needed to use `StyleSheet.flatten` there as well to make sure we can properly compare an array of objects (for example)